integration/python/src/test_double_spend.py

```diff
@@ -0,0 +1,41 @@
+# Double Spend testing
+# This test challenge the system with double spends.
```

Comment:
I don't have a line break in my docs, so it looks off since the first line has no punctuation at the end.
It's just a small detail, but if you touch the code you can check it again :)